schema/materials_category/structure.yaml

```diff
@@ -0,0 +1,107 @@
+material_category:
+  base:
+    - BaseConfiguration
+
+  single_material:
+    three_dimensional:
+      - supercell # No configuration, just a function
+
+    two_dimensional:
+      slab:
+        - Termination
+        - SlabConfiguration
+    #        - monolayer # Done in Notebooks
+
+    one_dimensional:
+      - nanotube # doesn't exist yet
+      - nanowire # doesn't exist yet
+
+    zero_dimensional:
+      nanoribbon:
+        - NanoribbonConfiguration
+        - EdgeTypes # Enum
```

Comment:
edge_types_enum

Reply:
Should not be at the same level as the configuration - should be in the schema directly